src/twig.core.js

```diff
@@ -1004,9 +1072,12 @@ var Twig = (function (Twig) {
             return Twig.Templates.registry[params.id];
         }
 
+        //if the parser name hasn't been set, default it to twig
+        params.parser = params.parser || 'twig';
+
         // Assume 'fs' if the loader is not defined
         loader = this.loaders[params.method] || this.loaders.fs;
-        return loader.apply(null, arguments);
+        return loader.apply(this, arguments);
     };
```

Comment:
I think we have to be careful with this line.  Since the loaders can be bound to a different scope during registration, applying to 'this' here, might not do what you expect.  It shouldn't affect the existing built-in loaders since they do not bind to a scope object, but if you look at the loader tests, it does.  In the tests, though, it does not take the parser into consideration, and so there are no problems.  But I thought I would point it out...